### DISCUSSION TITLE
Loss of resolution closing/opening a VDialog

### DISCUSSION THREAD

**Discussion by AliE89** [Kitware/trame#441] [General]
Hello Sebastien,

I am trying to pipe a view  inside a VDialog with VtkRemoteView(). 

The layout looks ok but:
- When I open the dialog, I need to click on the background to active the render (maybe some kind of update missing?)
- If I close the dialog and I re-open it up, I am losing the render resolution (got one a way lower). If I refresh the browser, than everything is back to normal.

Do you have any clue?

Thank you very much!

**Comment by jourdain** [comment]
You may have to trigger a "render/update" from the client once the dialog is totally visible. Did you check if the dialog trigger any event that you could leverage to trigger such render?

**Reply by AliE89** [reply]
Thanks,

What's the command to render/update? 

The only event I currently have, is a button click to open up the dialog, after that I was hoping to have the render ready. It works the first time (resolution is great), but it is not working the second time I click the button (the resolution is messed up and needs a page refresh).
I have tried this:

```
with vuetify.VRow(classes="pa-0 ma-0 d-flex", style=f"height: 1000px; width=200px"): #  
    with vuetify.VCol(classes="pa-0 ma-1"):
        html_view = pv_trame.VtkRemoteView(tmp_view, ref='test', interactive_ratio=1)
        html_view.update()
```
but no luck :(

**Reply by AliE89** [reply]
ps: this is the only event from VDialog according to docs:

![image](https://github.com/Kitware/trame/assets/49912570/18345d37-4f22-4bbe-98fe-505a96179627)

**Comment by jourdain** [comment]
The code snippet that you shared is indeed not correct and not what I meant. Could you share a small example reproducing the issue. Also if you want more help on your project, you can [reach out to Kitware for support](https://www.kitware.com/contact/).
